code/modules/mod/modules/modules_maint.dm

```diff
@@ -384,6 +399,13 @@
 	SIGNAL_HANDLER
 	speech_args[SPEECH_SPANS] |= "upside_down"
 
+/// Prevent someone from being buckled to the wearer while atrocinator is active
+/obj/item/mod/module/atrocinator/proc/on_someone_buckled(datum/source, mob/living/buckled_mob, mob/living/buckler)
+	SIGNAL_HANDLER
+	if(buckled_mob) // Someone is being buckled to the wearer
+		balloon_alert(mod.wearer, "atrocinator interference!")
+		return COMPONENT_BLOCK_BUCKLE
```

Comment:
The signal you use, `COMSIG_LIVING_SET_BUCKLED`, doesn't respond to this

You want `COMSIG_MOVABLE_PREBUCKLE`

Reply:
Also this balloon alert should go to the user trying to buckle, which could be the other guy in the case of piggybacking

Reply:
Also maybe just `balloon_alert("they're upside down!")`, that'd get the message across